src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav-list li {
+  margin-right: 20px;
+}
+
+.nav-list li:last-child {
+  margin-right: 0;
+}
+
+.nav-link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+a.is-active,
+a.is-active:hover,
+a.nav-link:hover,
+a.nav-link:active,
+a.is-active:active {
+  color: #00acdc;
+}
+
+a.is-active::after,
+a.is-active:active::after,
+a.nav-link:active::after {
+  content: "";
+  height: 4px;
+  width: 100%;
+  display: block;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
 }
```

Comment:
Don't use tags for styling, you need only to add this styles to `is-active` class